client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
According to the doc, these should be in alphabetical order, which makes it difficult to group them by the areas they correspond to. Either we should reconsider that requirement, or change our naming convention so that `FETCH` and `RECEIVE` come last.

@aduth what do you think?